packages/by-name/tdx-tools/package.nix

```diff
@@ -27,4 +28,12 @@ python3Packages.buildPythonApplication rec {
   preBuild = ''
     cd tests/lib/tdx-tools
   '';
+
+  meta = {
+    homepage = "https://github.com/canonical/tdx";
+    license = with lib.licenses; [
+      gpl3Only
+      asl20
+    ];
+  };
```

Comment:
General question: Should we set this for packages we aim to package within Contrast only?

Reply:
Yes, I find this helpful. You can click on the homepage link and license info is important for us, too. No need to add other meta fields like maintainers.